files/models.py

```
@@ -122,14 +133,18 @@ def topic_thumb_path(instance, filename):
     file_name = "{0}.{1}".format(friendly_token, helpers.get_file_name(filename))
     return settings.MEDIA_UPLOAD_DIR + "topics/{0}".format(file_name)
 
+
 def get_language_choices():
     """Get language choices dynamically to avoid database access during model import"""
     try:
-        return Language.objects.exclude(code__in=['automatic', 'automatic-translation']).values_list("code", "title")
+        return Language.objects.exclude(
+            code__in=["automatic", "automatic-translation"]
+        ).values_list("code", "title")
     except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)